canonical-websites/www.ubuntu.com
# Dynamic generation of navigation GA events

## Done

* Dynamically generate GA events for the following nav items:
  * global nav
  * primary nav
  * secondary nav
  * footer
  * contextual footer

**Event format:**
* category: `www.ubuntu.com-nav-<nav level>-<menu>` (eg. `www.ubuntu.com-nav-1-enterprise`)
* action: `from:<path> to:<path or external url>` (eg. `from:/ai to:/ai/consulting`)
* label: `<link text>` (eg. `Get started with Kubeflow`)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- In GA, look at the real-time events view in the combo tracker, filter on 'www.ubuntu.com-nav' and ensure click on nav items are registered.
